.github/workflows/rebuild-docker-image.yml

```diff
@@ -26,4 +26,4 @@ jobs:
           username: '${{ secrets.IOTALEDGER_HUB_DOCKER_LOGIN }}'
           password: '${{ secrets.IOTALEDGER_HUB_DOCKER_PASSWORD }}'
           tags: latest,${{github.event.inputs.tagName}}
-          tag_with_ref: true
\ No newline at end of file
+          build-args: DOWNLOAD_SNAPSHOT=0
```

Comment:
This implies that one will have to manually mount a snapshot as a volume, because none can be found in the image itself: sounds good.

Reply:
Yes. This workflow will be mostly used to rebuild the images from 0.7.5 to 0.8.3 that were not properly tagged during their release. So I think it makes sense that they should not include current snapshot.